src/components/App.tsx

```diff
@@ -5524,14 +5564,9 @@ class App extends React.Component<AppProps, AppState> {
                   },
                 },
                 this.scene.getNonDeletedElements(),
+                prevState,
               ),
             );
-          } else {
-            this.setState({
-              selectedElementIds: {},
-              selectedGroupIds: {},
-              editingGroupId: null,
-            });
```

Comment:
This (alongside the change below on 5555) feels like it should break something, but I couldn't find it.

What I'm solving by removing this is to not reset `selectedElementIds` to `{}` and right after set it to element ids within active box selection. This way it was always renewing the reference and thus during `pointermove` when selecting we'd not be able to memoize on the object.